deep_quoridor/src/quoridor.py

```diff
@@ -155,14 +191,17 @@ def is_wall_between(self, position1: Position, position2: Position) -> bool:
         assert position1_on_board or position2_on_board, "At least one position must be on the board"
 
         if position1_on_board and position2_on_board:
-            wall_position = position1 + position2
-            return self._grid[*wall_position] == Board.PLAYER1_WALL
+            wall_position = position1 + position2 + 2
+            return self._grid[*wall_position] == Board.WALL
         else:
             # By convention we treat the border as a "wall". This is makes checking jumps more convenient, since
             # players are allowed to jump diagonally if they are adjacent to another player and the border of the
             # board is on the other side of that player.
             return True
 
+    def get_old_style_walls(self):
```

Comment:
For now the Board class keeps track of the walls in both the new combined grid as well as the `(board_size-1) x (board_size -1) x 2` grid so that agents can use whichever they want.